tools/rawtools/rawtools.xml

```diff
@@ -94,6 +94,9 @@
     <inputs>
         <param format="thermo.raw" name="input_raws" type="data" label="Raw file" multiple="true"
             help="Select the raw file from history"/>
+        <param name="license_agreement" type="boolean" label="Do you agree to the vendor licenses?" help="This tool uses proprietary vendor libraries; to run it you must agree to the vendor licenses. Read them at http://proteowizard.sourceforge.net/licenses/thermo.html">
+            <validator type="expression" message="You must agree to the vendor licenses to run msconvert.">True == value</validator>
```

Comment:
You'll probably want to change the validator message and label to not point at ProteoWizard/msconvert. :) But having this validation on the client side is really nice. I wish more input validation could happen on the client side to avoid failed jobs.

Reply:
There are a lot of these validators and new once can be written :)